apps/docs/components/Breadcrumbs.tsx

```diff
@@ -109,15 +109,15 @@ const BreadcrumbsInternal = ({
                   </DrawerTrigger>
                   <DrawerContent showHandle={false}>
                     <div className="grid gap-1 px-4">
-                      {breadcrumbs
-                        .slice(1, -2)
-                        .map((crumb) =>
-                          crumb.url ? (
-                            <Link href={crumb.url}>{crumb.title || crumb.name}</Link>
-                          ) : (
-                            crumb.title || crumb.name
-                          )
-                        )}
+                      {breadcrumbs.slice(1, -2).map((crumb, index) =>
+                        crumb.url ? (
+                          <Link key={index} href={crumb.url}>
+                            {crumb.title || crumb.name}
+                          </Link>
+                        ) : (
+                          crumb.title || crumb.name
+                        )
+                      )}
```

Comment:
To resolve some React `key` errors. Ideally we'd use something like `nanoid()` instead of index, but not important for this PR.

Reply:
You found it, thank you!! 🙌🏼 🎉 🎺 ❤️ 

I've scanned that file twice now looking for this and somehow missed it 😄